### DISCUSSION TITLE
Regression Assignment

### DISCUSSION THREAD

**Discussion by pavittarx** [microsoft/ML-For-Beginners#266] [Q&A] [answered]
I have a couple of questions regarding this assignment and dataset. Secondly, I can't find solution for this assignment either.

Assignment: https://github.com/microsoft/ML-For-Beginners/blob/main/2-Regression/1-Tools/assignment.md
Lesson: https://github.com/microsoft/ML-For-Beginners/tree/main/2-Regression/1-Tools

I have examined the both the datasets. Please correct m if I am wrong somewhere.
- Diabetes dataset has 10 feature variables (X) and one target variable y.
- Linnerud Dataset has 3 feature variables (Weight, Waist, Pulse) and three target variables (Chins, Situps,  Jumps). 

The assignment asks to create a regression model b/w Waistline & No of situps accomplished. I need to know more how to create the models. 

Should I map, the single Waist feature variable to the Situps variable? or if to the three of them (targets)? or some relationships needs to be created between this other data?

**Comment by jlooper** [comment, marked as answer]
hi, the assignments don't have solutions, although I'm kind of wondering if having them available might be useful. Would they?

In general, the assignments allow you to make some decisions and have some creativity. They are supposed to help you explore the data to create a meaningful model. Why not try to map the relationship between waist and situps? I bet something logical can be discerned (...it's another example of moments where you don't have to have machine learning to create commonsense solutions, but still an interesting exercise) :)

**Reply by pavittarx** [reply]
Thanks Jen. 

I guess, if there are solutions as can have a look. Or if you don't want to provide complete solutions. Do drop some hints that would make it easier for us to verify, if we are going down the right lane. 

Although, from what I understand there is not always a perfect solution for anything. But, having one solution will help us wonder over if we are stuck, or just check that out of curiosity. 

Finally thanks for the explanation.
